server.js

```diff
@@ -13,10 +14,10 @@ new WebpackDevServer(webpack(config), {
   stats: {
     colors: true,
   },
-}).listen(4242, 'localhost', function (err) {
+}).listen(port, 'localhost', function (err) {
   if (err) {
     console.log(err);
   }
 
-  console.log('Listening at localhost:4242');
+  console.log(`Listening at localhost:${port}`);
```

Comment:
I don't want to overthink this, but trying your patch I thought of something that could be useful: displaying some instructions to change the port directly here.
Something like:
```javascript
if (port === 4242) {
  console.log('You can change this default port with the environment variable PERFHTML_PORT.').
}
```

What do you think ?